xbmc/addons/interfaces/GUI/Window.cpp

```diff
@@ -348,6 +696,130 @@ void* Interface_GUIWindow::get_list_item(void* kodiBase, void* handle, int listP
   return pItem;
 }
 
+void Interface_GUIWindow::set_current_list_position(void* kodiBase, void* handle, int list_position)
+{
+  CAddonDll* addon = static_cast<CAddonDll*>(kodiBase);
+  CGUIAddonWindow* pAddonWindow = static_cast<CGUIAddonWindow*>(handle);
+  if (!addon || !pAddonWindow)
+  {
+    CLog::Log(LOGERROR, "Interface_GUIWindow::%s - invalid handler data (kodiBase='%p', handle='%p') on addon '%s'",
+                          __FUNCTION__, addon, handle, addon ? addon->ID().c_str() : "unknown");
+    return;
+  }
+
+  Interface_GUIGeneral::lock();
+  pAddonWindow->SetCurrentListPosition(list_position);
```

Comment:
I see it vica versa - for C functions, I would use underline names, for C++ camel case. We're not consistent here in any way.

But may I suggest to do this kind of cleanup in a separate PR, once all stuff from Alwin is in? I think we agreed on this already.